middleware/security.test

```diff
@@ -8,20 +8,17 @@
 
```

Comment:
@brandondees @tmornini should this be called `policy` or `security`. The latter seems too vague in retrospect.